server/routes/auth.js

```diff
@@ -12,10 +12,13 @@ router.post("/login",passport.authenticate('local'), (req, res)=>{
     res.json(req.session.passport.user)
 })
 
+<<<<<<< HEAD
+=======
 router.post("/logout", (req, res)=>{
     req.logout();
     res.status(200).send({msg:"log out successfully"})
 })
+>>>>>>> 920bffcafe3212aaf55631370f5389931a4d6526
```

Comment:
This looks like you are committing a merge conflict. Please remove this line and also lines `15` and `16`